spec/requests/admin/participants_spec.rb

```diff
@@ -23,18 +23,22 @@
 
     it "only includes active participants" do
```

Comment:
So this is not true anymore no?

Reply:
It is not 😩

Reply:
I reworded the descriptions and re-pushed. Thanks @ltello